src/openscad.cc

```diff
@@ -817,37 +791,57 @@ int main(int argc, char **argv)
 
 	const char *output_file = nullptr;
 	const char *deps_output_file = nullptr;
+	
+	string colorSchemeNames;
+	for(const auto &name : ColorMap::inst()->colorSchemeNames()) {
+		if(!colorSchemeNames.empty())
+			colorSchemeNames.append(" | ");
+		if(name == ColorMap::inst()->defaultColorSchemeName())
+			colorSchemeNames.append("*");
+		colorSchemeNames.append(name);
+	}
 
+#ifdef ENABLE_EXPERIMENTAL
+	string features;
+	for(auto it = Feature::begin(); it != Feature::end(); ++it) {
+		if(!features.empty())
+			features.append(" | ");
+		features.append((*it)->get_name());
+	}
+#endif
+        
 	po::options_description desc("Allowed options");
 	desc.add_options()
-		("help,h", "help message")
+		("o,o", po::value<string>(), "output specified file instead of running the GUI, the file extension specifies the type: stl, off, amf, csg, dxf, svg, png, echo, ast, term, nef3, nefdbg\n")
+		("D,D", po::value<vector<string>>(), "var=val -pre-define variables")
+#ifdef ENABLE_EXPERIMENTAL
+		("p,p", po::value<string>(), "customizer parameter file")
+		("P,P", po::value<string>(), "customizer parameter set")
+		("enable", po::value<vector<string>>(), ("enable experimental features: " + features + "\n").c_str())
+#endif
+		("help,h", "print this help message and exit")
 		("version,v", "print the version")
-		("info", "print information about the building process")
-		("render", po::value<string>()->implicit_value(""), "if exporting a png image, do a full geometry evaluation")
-		("preview", po::value<string>()->implicit_value(""), "if exporting a png image, do an OpenCSG(default) or ThrownTogether preview")
-		("csglimit", po::value<unsigned int>(), "if exporting a png image, stop rendering at the given number of CSG elements")
-		("camera", po::value<string>(), "parameters for camera when exporting png")
-		("autocenter", "adjust camera to look at object center")
+		("info", "print information about the build process\n")
```

Comment:
Is the extra `\n` an oversight?

Reply:
No I deliberately separated all the png options.